examples/language-modeling/README.md

```diff
@@ -331,20 +331,64 @@ python3 run_lora_clm.py \
     --bf16 True \
     --output_dir ./model_lora_llama \
     --num_train_epochs 3 \
-    --per_device_train_batch_size 2 \
-    --per_device_eval_batch_size 2 \
-    --gradient_accumulation_steps 4 \
+    --per_device_train_batch_size 16 \
+    --gradient_accumulation_steps 1 \
     --evaluation_strategy "no" \
     --save_strategy "steps" \
-    --save_steps 2000 \
+    --max_steps 2418 \
```

Comment:
Just being curious, why this specific number of steps?

Reply:
this is the end of 3 epochs

Reply:
Why not using `--num_train_epochs 3` then?